test/go/iterative_verifier_test.go

```diff
@@ -1,477 +0,0 @@
-	t.Require().Nil(err)
-
-	result, err := t.verifier.VerifyDuringCutover()
-	t.Require().Nil(err)
-	t.Require().True(result.DataCorrect)
-	t.Require().Equal("", result.Message)
-}
-
-func (t *IterativeVerifierTestSuite) TestVerifyOnceWithIgnoredColumns() {
-	ignoredColumns := map[string]map[string]struct{}{"test_table_1": {"data": struct{}{}}}
-	t.verifier.IgnoredColumns = ignoredColumns
-
-	t.InsertRowInDb(42, "foo", t.Ferry.SourceDB)
-	t.InsertRowInDb(42, "bar", t.Ferry.TargetDB)
-
-	result, err := t.verifier.VerifyOnce()
-	t.Require().NotNil(result)
-	t.Require().Nil(err)
-	t.Require().True(result.DataCorrect)
-	t.Require().Equal("", result.Message)
-}
-
-func (t *IterativeVerifierTestSuite) TestVerifyOnceFails() {
-	t.InsertRowInDb(42, "foo", t.Ferry.SourceDB)
-	t.InsertRowInDb(42, "bar", t.Ferry.TargetDB)
-
-	result, err := t.verifier.VerifyOnce()
-	t.Require().NotNil(result)
-	t.Require().Nil(err)
-	t.Require().False(result.DataCorrect)
-	t.Require().Equal("verification failed on table: gftest.test_table_1 for pk: 42", result.Message)
-}
-
-func (t *IterativeVerifierTestSuite) TestVerifyCompressedOnceFails() {
-	t.InsertCompressedRowInDb(42, testhelpers.TestCompressedData1, t.Ferry.SourceDB)
-	t.InsertCompressedRowInDb(42, testhelpers.TestCompressedData2, t.Ferry.TargetDB)
-
-	result, err := t.verifier.VerifyOnce()
-	t.Require().NotNil(result)
-	t.Require().Nil(err)
-	t.Require().False(result.DataCorrect)
-	t.Require().Equal(
-		fmt.Sprintf("verification failed on table: %s.%s for pk: %s", testhelpers.TestSchemaName, testhelpers.TestCompressedTable1Name, "42"),
-		result.Message,
-	)
-}
-
-func (t *IterativeVerifierTestSuite) TestVerifyOncePass() {
-	t.InsertRowInDb(42, "foo", t.Ferry.SourceDB)
-	t.InsertRowInDb(42, "foo", t.Ferry.TargetDB)
-
-	result, err := t.verifier.VerifyOnce()
-	t.Require().NotNil(result)
-	t.Require().Nil(err)
-	t.Require().True(result.DataCorrect)
-	t.Require().Equal("", result.Message)
-}
-
-func (t *IterativeVerifierTestSuite) TestVerifyCompressedOncePass() {
-	t.InsertCompressedRowInDb(42, testhelpers.TestCompressedData1, t.Ferry.SourceDB)
-	t.InsertCompressedRowInDb(42, testhelpers.TestCompressedData1, t.Ferry.TargetDB)
-
-	result, err := t.verifier.VerifyOnce()
-	t.Require().NotNil(result)
-	t.Require().Nil(err)
-	t.Require().True(result.DataCorrect)
-	t.Require().Equal("", result.Message)
-}
-
-func (t *IterativeVerifierTestSuite) TestVerifyDifferentCompressedSameDecompressedDataOncePass() {
-	t.Require().NotEqual(testhelpers.TestCompressedData3, testhelpers.TestCompressedData4)
-
-	t.InsertCompressedRowInDb(43, testhelpers.TestCompressedData3, t.Ferry.SourceDB)
-	t.InsertCompressedRowInDb(43, testhelpers.TestCompressedData4, t.Ferry.TargetDB)
-
-	result, err := t.verifier.VerifyOnce()
-	t.Require().NotNil(result)
-	t.Require().Nil(err)
-	t.Require().True(result.DataCorrect)
-	t.Require().Equal("", result.Message)
-}
-
-func (t *IterativeVerifierTestSuite) TestBeforeCutoverFailuresFailAgainDuringCutover() {
```

Comment:
VerifyBeforeCutover doesn't exist anymore. This test is very granular as it calls into the IterativeVerifier directly. With the InlineVerifier, this test is covered via integration (calling Ghostferry from ruby):

https://github.com/Shopify/ghostferry/blob/b8147c5ff8653099f6c5e24128aa6fd41ff401e1/test/integration/inline_verifier_test.rb#L115-L138